src/main/java/embl/ebi/variation/eva/VariantJobsArgs.java

```diff
@@ -110,6 +117,7 @@
     private ObjectMap variantOptions  = new ObjectMap();
     private ObjectMap pipelineOptions  = new ObjectMap();
 
+    @PostConstruct
```

Comment:
Could this cause any conflicts when combined with invocations from test methods annotated with `@Before`? See for instance [the VariantsAnnotLoadTest class](https://github.com/EBIvariation/eva-pipeline/blob/develop/src/test/java/embl/ebi/variation/eva/pipeline/steps/VariantsAnnotLoadTest.java#L80).